build.gradle

```diff
@@ -9,9 +9,16 @@ buildscript {
 }
 
 allprojects {
+    apply plugin: 'java'
```

Comment:
…however, I don't like this change because it makes the information about plugins more indirect.

The core subproject would now have both `java` and `java-library`. Seems unintended.

Reply:
Let me think about this a little. There might be a better way for you.